ibm/service/scc/resource_ibm_scc_profile_attachment.go

```diff
@@ -716,23 +721,28 @@ func resourceIbmSccProfileAttachmentMapToAttachmentsPrototype(modelMap map[strin
 	return model, nil
 }
 
-func resourceIbmSccProfileAttachmentMapToMultiCloudScope(modelMap map[string]interface{}) (*securityandcompliancecenterapiv3.MultiCloudScope, error) {
-	model := &securityandcompliancecenterapiv3.MultiCloudScope{}
-	model.Environment = core.StringPtr(modelMap["environment"].(string))
-	properties := []securityandcompliancecenterapiv3.PropertyItem{}
+func resourceIbmSccProfileAttachmentMapToMultiCloudScope(modelMap map[string]interface{}) (*securityandcompliancecenterapiv3.MultiCloudScopePayload, error) {
+	model := &securityandcompliancecenterapiv3.MultiCloudScopePayload{}
+	properties := []securityandcompliancecenterapiv3.ScopePropertyIntf{}
 	for _, propertiesItem := range modelMap["properties"].([]interface{}) {
 		propertiesItemModel, err := resourceIbmSccProfileAttachmentMapToPropertyItem(propertiesItem.(map[string]interface{}))
 		if err != nil {
 			return model, err
 		}
-		properties = append(properties, *propertiesItemModel)
+		properties = append(properties, propertiesItemModel)
 	}
 	model.Properties = properties
+	if modelMap["environment"] != nil && modelMap["environment"].(string) != "" {
```

Comment:
Why are we doing only for environment as there can be other properties as well right ?

Reply:
Should be okay for now